parent/pom.xml

```diff
@@ -1513,7 +1505,19 @@
                <buildOutputDirectory>${basedir}/eclipse-output</buildOutputDirectory>
             </configuration>
          </plugin>
-
+         <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-checkstyle-plugin</artifactId>
+            <executions>
+               <execution>
+                  <id>checkstyle</id>
+                  <phase>validate</phase>
```

Comment:
Exactly.. the main though was that if it doesn't fit the rules, it's not worth compiling. What would be the argument for running it in `verify`?